.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: publish
+on:
+  schedule:
+    - cron: "0 0 * * 1" # midnight Monday UTC
+
+jobs:
+  publish:
+    name: publish
+    runs-on: ubuntu-16.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.PUBLISH_DEPLOY_KEY }}
+          fetch-depth: 0
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - name: Install cargo-workspaces
+        uses: actions-rs/install@v0.1
+        with:
+          crate: cargo-workspaces
+
+      - name: Release
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        shell: bash
+        run: |
+          git config --global user.name "Github Action"
+          cargo workspaces publish -ay --force '*' --include-merged-tags --exact minor
```

Comment:
Let's use long-hand format of options for `-ay`. Am I correct that this also would do a push with bumped version? I think we should not do this, versions at this repository should stay at `0.1.0` (really, it probably makes sense to flip them to `0.0.0`).

Reply:
Yeah, it does a bumped version push and tags the commit. I can not do the push but how would we know the next version to publish the next time this action is run?

Reply:
I'd just use the Action run ID as the version (the same we do for making releases).